### ISSUE TITLE
Password for changing wallet address - enhancement

### ISSUE THREAD

**Issue by Vipermarx** [nicehash/NiceHashMiner-Archived#1249]
Is it possible to have a configuration that you set password for adding and changing wallet address?

**Comment by NoSubstitute** [comment]
Good idea, but if someone already has access to your computer and login, you're pretty much ......
